evaluator/remediations.go

```diff
@@ -0,0 +1,28 @@
+package evaluator
+
+import (
+	"app/base"
+	"app/base/mqueue"
+	"github.com/RedHatInsights/patchman-clients/vmaas"
+)
+
+var remediationsPublisher = mqueue.WriterFromEnv("REMEDIATIONS_UPDATE_TOPIC")
```

Comment:
I would consider to make this sending optional. When REMEDIATIONS_UPDATE_TOPIC is empty, do not send anything. It can be useful for debugging and when running in testing environment when there is no consumer for these messages. Actually does it work in our local setup? Don't we need to create this topic in setup.sh?

Reply:
Fixed